src/main/java/com/kodilla/royal/game/of/ur/ComputerPlayer.java

```diff
@@ -1,15 +1,34 @@
 package com.kodilla.royal.game.of.ur;
 
+import java.util.Comparator;
+import java.util.PriorityQueue;
+import java.util.Queue;
+
 public class ComputerPlayer extends Player {
 
-    private final static int MAX_NO_OF_PIECES_ON_BOARD = 3;
+    public ComputerPlayer(
+            final Game game,
+            final int routeType,
+            final Field[][] fields
+    ) throws Exception {
+        super(PieceColor.LIGHT, game, routeType, fields);
+    }
+
+    public Piece chooseBestMove(int howFar) {
+        Comparator<Piece> pieceMoveComparator = Comparator.comparingInt(p -> p.getMovePriority(howFar, getRoute()));
```

Comment:
to moze byc statyczne pole w klasie, nie ma potrzeby za kazdym razem tworzyc nowego

Reply:
Ten comparator korzysta z lokalnej zmiennej `howFar` przekazywanej do metody `chooseBeatMove` jako parametr - przez to nie mam pomysłu jak zmienić go w statyczne pole klasy.